packages/@react-aria/radio/docs/useRadioGroup.mdx

```diff
@@ -113,7 +113,7 @@ function RadioGroup(props) {
         {children}
       </RadioContext.Provider>
       {description && <div {...descriptionProps} style={{fontSize: 12}}>{description}</div>}
-      {errorMessage && validationState === 'invalid' &&
+      {errorMessage && state.isInvalid &&
```

Comment:
need to keep validationState i think for backwards compact?

Reply:
`state.isInvalid` also takes `validationState` into account